sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/TestPodFactory.java

```diff
@@ -116,6 +113,30 @@ public static ResourceSet getResourceSet(String id, double cpu, double mem, doub
                 .build();
     }
 
+    public static PodSpec getMultiTaskPodSpec(
```

Comment:
Thought about that, but it may be useful for other test suites that want multi-task pods.